### PR TITLE
Update Eventwatcher docs to mention gitRepos have to be specified

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:
This PR adds descriptions on https://github.com/pipe-cd/pipe/pull/2452 and https://github.com/pipe-cd/pipe/pull/2454.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by nakabonne** [comment]
This should be merged right before releasing v0.16.0

File: docs/content/en/docs/operator-manual/piped/configuring-event-watcher.md [hunk 1]
```diff
@@ -11,24 +11,38 @@ To enable [EventWatcher](/docs/user-guide/event-watcher/), you have to configure
...
+If multiple Pipeds handle a single repository, you can prevent conflicts by splitting into the multiple EventWatcher files and setting `includes/excludes` to specify the files that should be monitored by this Piped.
+
+Say for instance, if you only want the Piped to use the Eventwatcher files under `.pipe/dev/`:
 
 ```yaml
 apiVersion: pipecd.dev/v1beta1
 kind: Piped
 spec:
   eventWatcher:
-    checkInterval: 5m
```
Comment (khanhtc1202) [inline docs/content/en/docs/operator-manual/piped/configuring-event-watcher.md:22]: nits, I think this configuration should remain in this snippet but it's up to you.

**Comment by khanhtc1202** [comment]
lgtm with a nits
/lgtm

File: docs/content/en/docs/operator-manual/piped/configuring-event-watcher.md [hunk 2]
```diff
@@ -11,24 +11,38 @@ To enable [EventWatcher](/docs/user-guide/event-watcher/), you have to configure
 ### [required] Grant write permission
 The [SSH key used by Piped](/docs/operator-manual/piped/configuration-reference/#git) must be a key with write-access because piped needs to commit and push to your git repository when any incoming event matches.
 
-### [optional] Settings for watcher
-The Piped's behavior can be finely controlled by setting the `eventWatcher` field.
+### [required] Specify Git repositories to be observed
```
Comment (nghialv) [inline docs/content/en/docs/operator-manual/piped/configuring-event-watcher.md:14]: I think by default, all tasks are considered as required so we can remove the `[required]` tag.
The optional tasks should include the `[optional]` tag explicitly.

File: docs/content/en/docs/operator-manual/piped/configuring-event-watcher.md (hunk 1 above)
Comment (nakabonne) [reply docs/content/en/docs/operator-manual/piped/configuring-event-watcher.md:22]: To emphasize specifying git repos, let me keep as is

File: docs/content/en/docs/operator-manual/piped/configuring-event-watcher.md (hunk 2 above)
Comment (nakabonne) [reply docs/content/en/docs/operator-manual/piped/configuring-event-watcher.md:14]: Makes sense. Let me do that

**Comment by nakabonne** [comment]
Applied. PTAL 😄

**Comment by nghialv** [comment]
Nice. 🏃‍♂️ 
/approve

File: docs/content/en/docs/operator-manual/piped/configuring-event-watcher.md [hunk 3]
```diff
@@ -8,27 +8,41 @@ description: >
...
 spec:
   eventWatcher:
-    checkInterval: 5m
     gitRepos:
       - repoId: repo-1
         commitMessage: Update values by Event watcher
         includes:
-          - event-watcher-dev.yaml
-          - event-watcher-stg.yaml
+          - dev/*.yaml
```
Comment (khanhtc1202) [inline docs/content/en/docs/operator-manual/piped/configuring-event-watcher.md:43]: just realized these values are relative paths under the `.pipe/` directory at root level of observing git repository. Should we explicitly mention it here?

Comment (nakabonne) [reply docs/content/en/docs/operator-manual/piped/configuring-event-watcher.md:43]: >Say for instance, if you only want the Piped to use the Eventwatcher files under `.pipe/dev/`:

At L32, it indicates this example is for `.pipe/dev/`. So I was guessing it's enough (a bit confusing though)

Comment (khanhtc1202) [reply docs/content/en/docs/operator-manual/piped/configuring-event-watcher.md:43]: okay, look like we note that users have to place their EventWatcher configuration files in `.pipe/` at root level from here so it should be ok 😇 
https://pipecd.dev/docs/user-guide/event-watcher/#1-defining-events

**Pull request merged**